version.php

```diff
@@ -30,7 +30,7 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = [26, 0, 2, 1];
+$OC_Version = [26, 0, 2, 2];
```

Comment:
revert to avoid conflict for release PR

```suggestion
$OC_Version = [26, 0, 2, 1];
```